packages/layout-grid/package.json

```diff
@@ -17,6 +17,8 @@
   "license": "MIT",
   "sass": "sass",
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
```

Comment:
ohhh we neeed to do this for every other package as well

Reply:
Yes will do Search+Replace

Reply:
Part of #1325